locustempus/urls.py

```diff
@@ -15,7 +15,7 @@
     auth_urls = url(r'^accounts/', include('djangowind.urls'))
 
 router = routers.DefaultRouter()
-router.register(r'projects', views.ProjectViewSet)
+router.register(r'projects', viewsets.ProjectApiView, basename='api-project')
```

Comment:
In general, these api endpoints are singular rather than plural. We can argue about this. 😸

Reply:
No need to argue! I agree singular makes more sense